docs/source/deployment/netlify.md

```diff
@@ -0,0 +1,188 @@
+Now, we can create a “Hello world” GraphQL API. Let’s put that in a new file in the lambda folder, at `src/lambda/graphql.js`:
+
+```js
+// src/lambda/graphql.js
+const { ApolloServer, gql } = require("apollo-server-lambda");
+
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`;
+
+const resolvers = {
+  Query: {
+    hello: (root, args, context) => {
+      return "Hello, world!";
+    }
+  }
+};
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers
+});
+
+exports.handler = server.createHandler();
+```
+
+Now, make sure you’ve run `npm run start:lambda`, and navigate to `localhost:9000/graphql` in your browser. You should see GraphQL Playground, where you can run queries against your API!
+
+<div style="text-align:center">
+![Local GraphQL Server](../images/graphql.png)
+<br></br>
+</div>
+
+If you can see GraphQL Playground and run a simple query, you’ve done everything properly! It’s pretty neat that getting an API up and running was just a few lines of code. Now let’s add Apollo to our frontend.
+
+### Adding Apollo Client to React
+
+Open up `src/App.js`. Let’s convert this file to load a simple query from the API we just set up. First, let’s install Apollo Client and the React integration, and we’ll use the `apollo-boost` package to get started easily:
+
+```bash
+npm install --save apollo-boost react-apollo
+```
+
+Now, add some code to initialize Apollo Client:
+
+```js
+// src/App.js
+import ApolloClient from "apollo-boost";
+const client = new ApolloClient({
+  uri: "/.netlify/functions/graphql"
+});
+```
+
+The way that the starter kit and Netlify are set up, any requests that start with `.netlify/functions` are automatically redirected to the Lambda functions in our project. So since we created our API with the filename `graphql.js`, we can call it with `.netlify/functions/graphql` from our frontend.
+
+Let’s replace the LambdaDemo component from the starter kit with a new one that uses Apollo to query our API:
+
+```js
+import { gql } from "apollo-boost";
+import { ApolloProvider, Query } from "react-apollo";
+// Replace the previous LambdaDemo with the code below:
+const LambdaDemo = () => (
+  <ApolloProvider client={client}>
+    <Query
+      query={gql`
+        {
+          hello
+        }
+      `}
+    >
+      {({ data }) =>
+        <div>A greeting from the server: {data.hello}</div>}
+    </Query>
+  </ApolloProvider>
+);
+```
+
+If we’ve got everything set up correctly, we’ll see the greeting that we wrote in our resolver show up in the frontend:
+
+<div style="text-align:center">
+![React Frontend](../images/welcometoreact.png)
+<br></br>
+</div>
+
+Sweet, now we’ve connected our client to our API, and the only thing left is to deploy it! We can deploy both our frontend and API to Netlify in just one step.
+
+### Deploying to Netlify
+
+The best way to deploy to Netlify is to set it up to deploy automatically from a GitHub repository. Therefore, create a new repository and push the code there.
```

Comment:
I'm not sure we're in a position to say what's best for Netlify.  Can we word this differently?

> Netlify supports automatically deploying from a GitHub repository. Therefore, ...